services/common/src/components/projectSummary/FacilityOperator.tsx

```diff
@@ -17,10 +17,39 @@ import { getDropdownProvinceOptions } from "@mds/common/redux/selectors/staticCo
 import RenderRadioButtons from "../forms/RenderRadioButtons";
 import RenderAutoSizeField from "../forms/RenderAutoSizeField";
 import { normalizePhone } from "@mds/common/redux/utils/helpers";
+import { getRegionOptions } from "@mds/common/redux/slices/regionsSlice";
 
 export const FacilityOperator: FC = () => {
   const formValues = useSelector(getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY));
-  const { zoning } = formValues;
+
+  const formErrors = useSelector(getFormSyncErrors(FORM.ADD_EDIT_PROJECT_SUMMARY));
+  const {
+    facility_coords_source,
+    zoning,
+    facility_latitude,
+    facility_longitude,
+    legal_land_desc,
+    facility_pid_pin_crown_file_no,
+  } = formValues;
```

Comment:
I can see where the lat & lon are being used in this file, but are all the other destructured attributes being used?

Reply:
Hmmm, good question.  A bunch of this stuff was updated in the rebase.  I'll take a look.